cygnus-common/src/main/java/com/telefonica/iot/cygnus/backends/mongo/MongoBackendImpl.java

```diff
@@ -107,7 +107,7 @@ public void createCollection(String dbName, String collectionName, long dataExpi
         try {
             db.createCollection(collectionName);
         } catch (Exception e) {
-            if (e.getMessage().contains("collection already exists")) {
+            if (e.getMessage().contains("\"code\" : 48")) {
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1388/commits/0a993cea984fc2b368af697da91ebf441a59bdf6